deployment/cloud-run/model_utils.py

```
@@ -1,254 +1,244 @@
+        if not isinstance(text, str):
+            results[i] = {
+                'error': 'Text must be a non-empty string',
+                'emotions': [],
+                'confidence': 0.0
+            }
+        elif not text.strip():
+            results[i] = {
+                'error': 'Text must be a non-empty string',
+                'emotions': [],
+                'confidence': 0.0
+            }
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)